security/pkg/server/ca/server.go

```diff
@@ -272,14 +272,12 @@ func (s *Server) authenticate(ctx context.Context) *authenticate.Caller {
 	// TODO: apply different authenticators in specific order / according to configuration.
 	for _, authn := range s.authenticators {
 		u, err := authn.Authenticate(ctx)
-		if err != nil {
-			log.Warnf("Authentication failed: %v", err)
-		}
 		if u != nil && err == nil {
-			log.Infof("Authentication successful through auth source %v", u.AuthSource)
+			log.Debugf("Authentication successful through auth source %v", u.AuthSource)
 			return u
 		}
 	}
+	log.Warnf("Authentication failed")
```

Comment:
I see, it moved here.  Any chance we could get the error details back into the log message?

Reply:
Seems not. I will change the removed lines to Debugf. That can help.